index.css

```diff
@@ -0,0 +1,310 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.header1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: auto;
+  padding: 30px 0 15px 0;
+}
+
+.header1 h3 {
+  font-weight: 700;
+  font-style: bold;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.header1 h3 a {
+  text-decoration: none;
+  color: #6070ff;
+  font-size: 18px;
+}
+
+.section1 {
+  background: url(./img//background1.png);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  padding: 100px 0;
+}
+
+.section1 .intro {
+  width: 90%;
+  margin: auto;
+}
+
+.section1 h1 {
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  align-items: left;
+  font-weight: 800;
+  width: 290px;
+  font-style: bold;
+}
+
+.section1 p {
+  color: #344563;
+  font-size: 16px;
+  line-height: 24px;
+  width: 300px;
+  font-weight: 500;
+  align-items: left;
+  padding: 50px 0 0 0;
+}
+
+.socialmedia_connect h5 {
+  display: flex;
+  flex-direction: row;
+  font-size: 16px;
+  line-height: 24px;
+  align-items: left;
+  color: #7f8cff;
+  font-weight: 500;
+  padding-top: 15px;
+}
+
+.socialmedia_connect ul {
+  display: flex;
+  gap: 20px;
+  align-items: left;
+  list-style: none;
+  padding-top: 15px;
+  margin-bottom: 50px;
+}
+
+.socialmedia_connect ul li {
+  text-decoration: none;
+}
+
+.works_section {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 10px;
+  width: 90%;
+  margin: auto;
+  padding: 20px 0 30px 0;
+}
+
```

Comment:
- [ ] Your projects look great on 375px, but when the width is bigger the responsive until desktop version 768 px it's lost. To fix this we can improve your article class given a `display: flex;
 flex-direction: column;
 align-items: center;`

In this way, you will have your card's article centered in the middle whatever the width of the view is.

***This is how your cards look until with a wide 375px or more***

![image](https://user-images.githubusercontent.com/93630700/205315878-5aaa6784-d16c-41c1-ada5-12b5fd2e244e.png)


***Here it's your card with the property is that I mention at the beginning*** 💯💯💯💯 

![image](https://user-images.githubusercontent.com/93630700/205315600-ad0c6f19-a5cf-4c6b-8195-71029c4c6ccf.png)